dashboards/f5/f5-big-ip.json

```diff
@@ -0,0 +1,1184 @@
+                }
+              ],
+              "platformOptions": {
+                "ignoreTimeRange": false
+              },
+              "yAxisLeft": {
+                "max": 100,
+                "min": 0,
+                "zero": false
+              }
+            }
+          },
+          {
+            "title": "Memory %",
+            "layout": {
+              "column": 10,
+              "row": 4,
+              "width": 3,
+              "height": 4
+            },
+            "linkedEntityGuids": null,
+            "visualization": {
+              "id": "viz.line"
+            },
+            "rawConfiguration": {
+              "facet": {
+                "showOtherSeries": false
+              },
+              "legend": {
+                "enabled": true
+              },
+              "nrqlQueries": [
+                {
+                  "accountIds": [],
+                  "query": "FROM F5BigIpSystemSample SELECT\n  ( average(system.memoryUsedInBytes) / average(system.memoryTotalInBytes) )*100 AS 'Memory %'\nFACET displayName AS 'F5 System'\nWHERE displayName IN ( {{f5_system}} )\nTIMESERIES\nLIMIT MAX"
+                }
+              ],
+              "platformOptions": {
+                "ignoreTimeRange": false
+              },
+              "yAxisLeft": {
+                "max": 100,
+                "min": 0,
+                "zero": false
+              }
+            }
+          }
+        ]
+      },
+      {
+        "name": "Nodes",
+        "description": null,
+        "widgets": [
+          {
+            "title": "",
+            "layout": {
+              "column": 1,
+              "row": 1,
+              "width": 3,
+              "height": 4
+            },
+            "linkedEntityGuids": null,
+            "visualization": {
+              "id": "viz.markdown"
+            },
+            "rawConfiguration": {
+              "text": "![f5 logo](https://sjc04pap001files.storage.live.com/y4mmTKKQgYW0voEkAP7b-jMes8Eoopinjpvx1iMaO0omXrOF6mG9nlRWai-CilsMmC6UT_KTGYpCNmIlMqHBYIx5VQgSWKSCcF0YVcw8zH8iRsYWNkqmu-rljXIVkaG-2WooObwwbKZmYCILlEMIMqxRZLtvp2UMOjgpG-DlwXcuYR3IodghiBh-br0MMMgB0qvnkYJK6weLCzvRJc5i4jVoQ?encodeFailures=1&width=110&height=110)\n\n### Nodes\nA [node](https://techdocs.f5.com/kb/en-us/products/big-ip_ltm/manuals/product/ltm-concepts-11-5-1/5.html) is a logical object on the BIG-IP® Local Traffic Manager™ system that identifies the IP address of a physical resource on the network. You can explicitly create a node, or you can instruct Local Traffic Manager (LTM®) to automatically create one when you add a pool member to a load balancing pool.\n\nThe difference between a node and a pool member is that a node is designated by the device’s IP address only (`10.10.10.10`), while designation of a pool member includes an IP address and a service (such as `10.10.10:8`)."
```

Comment:
Lets also update this url to a new relic managed one
https://newrelic.com/sites/default/files/quickstarts/images/icons/f5--logo.svg